packages/backend-core/scripts/test.sh

```diff
@@ -4,10 +4,10 @@ set -e
 if [[ -n $CI ]]
 then
   # --runInBand performs better in ci where resources are limited
-  echo "jest --coverage --runInBand --forceExit"
-  jest --coverage --runInBand --forceExit
+  echo "jest --coverage --runInBand --forceExit $@"
+  jest --coverage --runInBand --forceExit $@
 else
   # --maxWorkers performs better in development
-  echo "jest --coverage --detectOpenHandles"
-  jest --coverage --detectOpenHandles
+  echo "jest --coverage --forceExit --detectOpenHandles $@"
```

Comment:
Why is this needed? Don't we actually want to detect the handles?

Reply:
Sorry, I meant the `--forceExit`

Reply:
Ohh sorry, I ran these tests once and they got stuck exiting because of an open Redis client. I did this out of laziness and because it's done elsewhere.